proxy/proxy.go

```diff
@@ -88,6 +91,22 @@ func (d *Proxy) RoundTrip(r *http.Request) (*http.Response, error) {
 			Header:     rw.header,
 		}, nil
 	} else if err == nil {
+		if d.t.IsLoaded() {
+			var sp opentracing.Span
+			spanName := "forwarding_to:" + r.URL.String()
```

Comment:
Given that this would be many, many operations depending on the number of URLs, could it be a problem with cardinality? I was under the impression that one would use [semantic conventions for tags](https://opentracing.io/specification/conventions/) such as `http.url` and a span name which mimics the function name (in this case `round_trip` or something similar)

Reply:
you're right about semantic convention : if there's a standard, it's better to follow it.
about how to implement that, traefik has tracing capabilities so I'll have a look about how they did that and come back to you.

Reply:
Thank you, that would be really helpful!